broker/broker.go

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Mainflux
+// SPDX-License-Identifier: Apache-2.0
+
+package broker
+
+import (
+	"context"
+
+	"github.com/nats-io/nats.go"
+)
+
+// SubjectAllChannels allows to subscribe to all subjects of all channels
+const SubjectAllChannels = "channel.>"
+
+// Nats specifies a NATS message API.
+type Nats interface {
```

Comment:
You think its better to have a separate interface for each broker that we use like `nats` `kafka` etc...? Not one Interface lets say `Messenger` that needs to be satisfied by broker imply for example `nats`.

Reply:
Agree.

Reply:
My first idea is to have a single interface. But it's not possible unless you create a new service for this because NATS interface will never match with Kafka one. For now if we want to use this as package it's better to have 2 interfaces.